server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoErrorRetryAnalyzer.ts

```diff
@@ -43,23 +43,12 @@ export class MongoErrorRetryAnalyzer {
 
 	public shouldRetry(error: Error): boolean {
 		const rule = this.getRetryRule(error);
-		const sanitizedError = {
```

Comment:
The `error` from this context is already sanitized, no need to sanitize again and causing logs hard to read